projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/general/unbreakable.json

```
@@ -0,0 +1,14 @@
+{
+  "modifier_id": "tconstruct:unbreakable",
+  "text": [
+    {
+      "text": "毁坏的事物是无趣的，但谢天谢地有不毁的事物存在。"
```

```suggestion
      "text": "装备坏了可并不好笑，谢天谢地你可以把它们做成不毁。"
```
not fun翻成并不好笑会好一些，机翻味没那么重。

---

做成不毁的